cmd/http-tracer.go

```diff
@@ -32,6 +32,8 @@ import (
 	trace "github.com/minio/minio/pkg/trace"
 )
 
+var bodyPlaceHolder = []byte("<BODY>")
```

Comment:
`traceBodyPlaceHolder` would be better - to scope the name

Reply:
@krishnasrinivas done.